diff --git a/src/test/java/org/json/junit/XMLTest.java b/src/test/java/org/json/junit/XMLTest.java
index 2c8f0d1..3e5a7b2 100644
--- a/src/test/java/org/json/junit/XMLTest.java
+++ b/src/test/java/org/json/junit/XMLTest.java
@@ -1178,15 +1178,15 @@ public class XMLTest {
     public void testIndentSimpleJsonObject(){
         String str = "{    \"employee\": {  \n" +
                 "        \"name\":       \"sonoo\",   \n" +
-                "        \"salary\":      56000,   \n" +
+                "        \"salary\":      56000,   \n" +
                 "        \"married\":    true  \n" +
                 "    }}";
         JSONObject jsonObject = new JSONObject(str);
         String actual = XML.toString(jsonObject, "Test", 2);
         String expected = "<Test>\n" +
                 "  <employee>\n" +
                 "    <name>sonoo</name>\n" +
-                "    <salary>56000</salary>\n" +
+                "    <salary>56000</salary>\n" +
                 "    <married>true</married>\n" +
                 "  </employee>\n" +
                 "</Test>\n";
         assertEquals(actual, expected);
     }
