diff --git a/src/test/java/us/bpsm/edn/printer/PrinterTest.java b/src/test/java/us/bpsm/edn/printer/PrinterTest.java
index 1234567..89abcde 100644
--- a/src/test/java/us/bpsm/edn/printer/PrinterTest.java
+++ b/src/test/java/us/bpsm/edn/printer/PrinterTest.java
@@ -111,7 +111,7 @@
     public void testPrettyPrinting() {
         Map<Integer, String> m = new HashMap();
         m.put(3, "Three");
-        m.put(4, "Four");
+        m.put(4, "Four");
         List<?> list = Arrays.asList(new HashSet(Arrays.asList(1, 2)), m);
         String s = Printers.printString(Printers.prettyPrinterProtocol(), list);
         assertEquals("[\n  #{\n    1\n    2\n  }\n  {\n    3 \"Three\"\n    4 \"Four\"\n  }\n]", s);
