diff --git a/src/test/java/net/sf/marineapi/ais/event/AbstractAISMessageListenerTest.java b/src/test/java/net/sf/marineapi/ais/event/AbstractAISMessageListenerTest.java
index 1234567..89abcde 100644
--- a/src/test/java/net/sf/marineapi/ais/event/AbstractAISMessageListenerTest.java
+++ b/src/test/java/net/sf/marineapi/ais/event/AbstractAISMessageListenerTest.java
@@ -118,10 +118,10 @@ public class AbstractAISMessageListenerTest {
     public void testGenericsListenerDefaultConstructorThrows() {
         try {
             GenericsListener<Integer, AISMessage01> gl = new GenericsListener<>();
-            fail("exception not thrown, resolved to " + gl.messageType);
+            fail("exception not thrown, resolved to " + gl.messageType);
         } catch (IllegalStateException ise) {
             assertEquals("Cannot resolve generic type <T>, use constructor with Class<T> param.", ise.getMessage());
         } catch (Exception e) {
-            fail("unexpected exception thrown: " + e.getMessage());
+            fail("unexpected exception thrown: " + e.getMessage());
         }
     }
 }