diff --git a/core/src/test/java/com/pholser/junit/quickcheck/internal/generator/ExplicitGroupOfPropertyParameterGeneratorsChosenWithDiscreteProbabilityTest.java b/core/src/test/java/com/pholser/junit/quickcheck/internal/generator/ExplicitGroupOfPropertyParameterGeneratorsChosenWithDiscreteProbabilityTest.java
index 3b0f0d1..e8b0f0d 100644
--- a/core/src/test/java/com/pholser/junit/quickcheck/internal/generator/ExplicitGroupOfPropertyParameterGeneratorsChosenWithDiscreteProbabilityTest.java
+++ b/core/src/test/java/com/pholser/junit/quickcheck/internal/generator/ExplicitGroupOfPropertyParameterGeneratorsChosenWithDiscreteProbabilityTest.java
@@ -25,7 +25,7 @@ public class ExplicitGroupOfPropertyParameterGeneratorsChosenWithDiscreteProbab
     @Override protected void primeSourceOfRandomness() {
         when(randomForParameterGenerator.nextInt(6)).thenReturn(0).thenReturn(1).thenReturn(5);
     }
-
+    @Override protected int trials() {
+        return 3;
+    }
 
     @Override protected List<?> randomValues() {
         return asList("foo", "bar", "baz");
