diff --git a/core/src/test/java/tk/mybatis/mapper/mapperhelper/ComplexEntityTest.java b/core/src/test/java/tk/mybatis/mapper/mapperhelper/ComplexEntityTest.java
index 1234567..89abcde 100644
--- a/core/src/test/java/tk/mybatis/mapper/mapperhelper/ComplexEntityTest.java
+++ b/core/src/test/java/tk/mybatis/mapper/mapperhelper/ComplexEntityTest.java
@@ -154,13 +154,13 @@ public void test() {
         final String sql = sqlBuilder.toString();
         Assert.assertEquals("SELECT id,user_name,address,state  FROM user " +
                 "<where>" +
-                "<if test=\"id != null\"> AND id = #{id}</if>" +
-                "<if test=\"userName != null\"> AND user_name = #{userName}</if>" +
-                "<if test=\"address != null\"> AND address = #{address, typeHandler=tk.mybatis.mapper.mapperhelper.ComplexEntityTest.AddressHandler}</if>" +
-                "<if test=\"state != null\"> AND state = #{state}</if></where>", sql);
+                "<if test=\"id != null\"> AND id = #{id}</if>" +
+                "<if test=\"userName != null\"> AND user_name = #{userName}</if>" +
+                "<if test=\"address != null\"> AND address = #{address, typeHandler=tk.mybatis.mapper.mapperhelper.ComplexEntityTest.AddressHandler}</if>" +
+                "<if test=\"state != null\"> AND state = #{state}</if></where>", sql);
 
         final ResultMap resultMap = entityTable.getResultMap(configuration);
         final List<ResultMapping> resultMappings = resultMap.getResultMappings();
@@ -164,19 +164,19 @@ public void test() {
         final ResultMapping stateMapping = resultMappings.get(3);
 
         Assert.assertEquals("id", idMapping.getColumn());
-        Assert.assertEquals("id", idMapping.getProperty());
-        Assert.assertTrue(idMapping.getFlags().contains(ResultFlag.ID));
+        Assert.assertEquals("id", idMapping.getProperty());
+        Assert.assertTrue(idMapping.getFlags().contains(ResultFlag.ID));
 
         Assert.assertEquals("user_name", userNameMapping.getColumn());
-        Assert.assertEquals("userName", userNameMapping.getProperty());
+        Assert.assertEquals("userName", userNameMapping.getProperty());
 
         Assert.assertEquals("address", addressMapping.getColumn());
-        Assert.assertEquals("address", addressMapping.getProperty());
-        Assert.assertEquals(AddressHandler.class, addressMapping.getTypeHandler().getClass());
+        Assert.assertEquals("address", addressMapping.getProperty());
+        Assert.assertEquals(AddressHandler.class, addressMapping.getTypeHandler().getClass());
 
         Assert.assertEquals("state", stateMapping.getColumn());
-        Assert.assertEquals("state", stateMapping.getProperty());
-        Assert.assertEquals(EnumTypeHandler.class, stateMapping.getTypeHandler().getClass());
+        Assert.assertEquals("state", stateMapping.getProperty());
+        Assert.assertEquals(EnumTypeHandler.class, stateMapping.getTypeHandler().getClass());
