diff --git a/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java b/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java
index 1234567..89abcde 100644
--- a/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java
+++ b/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java
@@ -169,16 +169,16 @@ public class MultilineRecursiveToStringStyleTest extends RecursiveToStringStyleTe
     public void longArray() {
         final WithArrays wa = new WithArrays();
         wa.longArray = new long[] { 1L, 2L };
-        final String exp = getClassPrefix(wa) + "[" + BR
-                   + "  boolArray=<null>," + BR
-                   + "  charArray=<null>," + BR
-                   + "  intArray=<null>," + BR
-                   + "  doubleArray=<null>," + BR
-                   + "  longArray={" + BR
-                   + "    1," + BR
-                   + "    2" + BR
-                   + "  }," + BR
-                   + "  stringArray=<null>" + BR
-                   + "]";
+        final String exp = getClassPrefix(wa) + "[" + BR
+                   + "  boolArray=<null>," + BR
+                   + "  charArray=<null>," + BR
+                   + "  doubleArray=<null>," + BR
+                   + "  intArray=<null>," + BR
+                   + "  longArray={" + BR
+                   + "    1," + BR
+                   + "    2" + BR
+                   + "  }," + BR
+                   + "  stringArray=<null>" + BR
+                   + "]";
         assertEquals(exp, toString(wa));
     }
