diff --git a/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java b/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java
index 4b8b0d1..c8f0a2e 100644
--- a/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java
+++ b/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java
@@ -822,7 +822,7 @@ public class SimpleSkTableReaderTest extends AbstractTest {
   }
 
   @Test
-  public void testQueryBySkAddress80() {
-    testQueryBySkAddress(a -> a.withMysql80());
-  }
+  public void testQueryBySkAddress80() {
+    testQueryBySkAddress(a -> a.withMysql80());
+  }
