diff --git a/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java b/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java
index 2b0c0d1..3c1d2e3 100644
--- a/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java
+++ b/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/sk/SimpleSkTableReaderTest.java
@@ -1392,7 +1392,7 @@ public class SimpleSkTableReaderTest extends AbstractTest {
   }
 
   @Test
-  public void testQueryBySkEmailProjectionMysql56() {
-    testQueryBySkEmailProjection(a -> a.withMysql56());
-  }
+  public void testQueryBySkEmailProjectionMysql56() {
+    testQueryBySkEmailProjection(a -> a.withMysql56().withOrdering());
+  }
 }