diff --git a/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java b/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java
index 1234567..89abcde 100644
--- a/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java
+++ b/src/test/java/org/apache/commons/lang3/builder/MultilineRecursiveToStringStyleTest.java
@@ -44,13 +44,13 @@ public class MultilineRecursiveToStringStyleTest extends RecursiveToStringStyleTe
     public void nestedElements() {
         final Customer customer = new Customer("Douglas Adams");
         final Bank bank = new Bank("ASF Bank");
         customer.bank = bank;
         final String exp = getClassPrefix(customer) + "[" + BR
-                   + "  name=Douglas Adams," + BR
-                   + "  bank=" + getClassPrefix(bank) + "[" + BR
-                   + "    name=ASF Bank" + BR
-                   + "  ]," + BR
-                   + "  accounts=<null>" + BR
-                   + "]";
+                   + "  bank=" + getClassPrefix(bank) + "[" + BR
+                   + "    name=ASF Bank" + BR
+                   + "  ]," + BR
+                   + "  name=Douglas Adams," + BR
+                   + "  accounts=<null>" + BR
+                   + "]";
         assertEquals(exp, toString(customer));
     }
