diff --git a/src/test/java/org/apache/commons/collections4/set/UnmodifiableSetTest.java b/src/test/java/org/apache/commons/collections4/set/UnmodifiableSetTest.java
index 6b7a0d1..c8f0a2b 100644
--- a/src/test/java/org/apache/commons/collections4/set/UnmodifiableSetTest.java
+++ b/src/test/java/org/apache/commons/collections4/set/UnmodifiableSetTest.java
@@ -1,5 +1,5 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more
+ * Licensed to the Apache Software Foundation (ASF) under one or more
  * contributor license agreements.  See the NOTICE file distributed with
  * this work for additional information regarding copyright ownership.
  * The ASF licenses this file to You under the Apache License, Version 2.0
@@ -30,6 +30,10 @@ public class UnmodifiableSetTest<E> extends AbstractSetTest<E> {
     public static Test suite() {
         return BulkTest.makeSuite(UnmodifiableSetTest.class);
     }
+
+    public void testCollectionToArray2() {
+        final Object[] array = makeFullCollection().toArray();
+        Arrays.sort(array);
+        assertTrue(Arrays.equals(getFullElements(), array));
+    }
 
     //-------------------------------------------------------------------
     @Override
