diff --git a/src/test/java/net/sf/marineapi/ais/parser/AISMessageFactoryTest.java b/src/test/java/net/sf/marineapi/ais/parser/AISMessageFactoryTest.java
index 8b0c0d1..e4f5a2c 100644
--- a/src/test/java/net/sf/marineapi/ais/parser/AISMessageFactoryTest.java
+++ b/src/test/java/net/sf/marineapi/ais/parser/AISMessageFactoryTest.java
@@ -26,6 +26,9 @@ public class AISMessageFactoryTest {
     private AISMessageFactory amf;
     private AISSentence single;
 
+    @Before
+    public void setUp() {
+        amf = new AISMessageFactory();
+    }
+
     @Test
     public void testCreate() {
         AISMessage msg = amf.create(single);
         assertTrue(msg instanceof AISMessage01);
         assertEquals(1, msg.getMessageType());
     }
