diff --git a/core/src/test/java/com/pholser/junit/quickcheck/SamplingButAlsoIncludingAGivenSetTest.java b/core/src/test/java/com/pholser/junit/quickcheck/SamplingButAlsoIncludingAGivenSetTest.java
--- a/core/src/test/java/com/pholser/junit/quickcheck/SamplingButAlsoIncludingAGivenSetTest.java
+++ b/core/src/test/java/com/pholser/junit/quickcheck/SamplingButAlsoIncludingAGivenSetTest.java
@@ -249,6 +249,8 @@
     }
 
     @Test public void wrapperFloats() throws Exception {
+        WrapperFloats.iterations = 0;
+        WrapperFloats.values.clear();
         assertThat(testResult(WrapperFloats.class), isSuccessful());
         assertEquals(defaultPropertyTrialCount(), WrapperFloats.iterations);
         assertEquals(
