diff --git a/src/test/java/com/adyen/CheckoutTest.java b/src/test/java/com/adyen/CheckoutTest.java
--- a/src/test/java/com/adyen/CheckoutTest.java
+++ b/src/test/java/com/adyen/CheckoutTest.java
@@ -1445,7 +1445,7 @@
         paymentsRequest.setPaymentMethod(payUUpiDetails);
 
         String gson = GSON.toJson(paymentsRequest);
-        assertEquals(expectedJson, gson);
+        assertEquals(GSON.fromJson(expectedJson, Object.class), GSON.fromJson(gson, Object.class));
 
         String jackson = OBJECT_MAPPER.writeValueAsString(paymentsRequest);
         assertEquals(expectedJson, jackson);
