diff --git a/src/main/java/org/apache/commons/lang3/reflect/FieldUtils.java b/src/main/java/org/apache/commons/lang3/reflect/FieldUtils.java
--- a/src/main/java/org/apache/commons/lang3/reflect/FieldUtils.java
+++ b/src/main/java/org/apache/commons/lang3/reflect/FieldUtils.java
@@ -218,6 +218,7 @@
         Class<?> currentClass = cls;
         while (currentClass != null) {
             final Field[] declaredFields = currentClass.getDeclaredFields();
+            java.util.Arrays.sort(declaredFields, java.util.Comparator.comparing(java.lang.reflect.Field::getName));
             Collections.addAll(allFields, declaredFields);
             currentClass = currentClass.getSuperclass();
         }
