diff --git a/core/src/test/java/com/pholser/junit/quickcheck/PropertyParameterConfigurationIsolationTest.java b/core/src/test/java/com/pholser/junit/quickcheck/PropertyParameterConfigurationIsolationTest.java
--- a/core/src/test/java/com/pholser/junit/quickcheck/PropertyParameterConfigurationIsolationTest.java
+++ b/core/src/test/java/com/pholser/junit/quickcheck/PropertyParameterConfigurationIsolationTest.java
@@ -110,6 +110,7 @@
 
     @Test public void acrossParametersOfParametersOfParameterizedTypeAndUnparameterizedTypeWithOneConstant()
         throws Exception {
+        ParametersOfParameterizedTypeAndUnparameterizedTypeWithOneConstant.iterations = 0;
 
         assertThat(
             testResult(ParametersOfParameterizedTypeAndUnparameterizedTypeWithOneConstant.class),
