diff --git a/src/test/java/com/adyen/CheckoutTest.java b/src/test/java/com/adyen/CheckoutTest.java
--- a/src/test/java/com/adyen/CheckoutTest.java
+++ b/src/test/java/com/adyen/CheckoutTest.java
@@ -1208,10 +1208,10 @@
         paymentsRequest.setPaymentMethod(visaCheckoutDetails);
 
         String gson = GSON.toJson(paymentsRequest);
-        assertEquals(expectedJson, gson);
-
-        String jackson = OBJECT_MAPPER.writeValueAsString(paymentsRequest);
-        assertEquals(expectedJson, jackson);
+        assertEquals(GSON.fromJson(expectedJson, Object.class), GSON.fromJson(gson, Object.class));
+
+        String jackson = OBJECT_MAPPER.writeValueAsString(paymentsRequest);
+        assertEquals(GSON.fromJson(expectedJson, Object.class), GSON.fromJson(jackson, Object.class));
     }
 
     @Test
