diff --git a/src/test/java/com/alibaba/json/bvt/issue_2400/Issue2430.java b/src/test/java/com/alibaba/json/bvt/issue_2400/Issue2430.java
--- a/src/test/java/com/alibaba/json/bvt/issue_2400/Issue2430.java
+++ b/src/test/java/com/alibaba/json/bvt/issue_2400/Issue2430.java
@@ -17,8 +17,7 @@
 		vo.setMap(multimap);
 		vo.setName("zhangsan");
 
-		assertEquals("{\"map\":{\"a\":[\"1\",\"2\",\"3\"],\"b\":[\"1\"]},\"name\":\"zhangsan\"}",
-				JSON.toJSONString(vo));
+		assertEquals(com.alibaba.fastjson.JSON.parse("{\"map\":{\"a\":[\"1\",\"2\",\"3\"],\"b\":[\"1\"]},\"name\":\"zhangsan\"}"), com.alibaba.fastjson.JSON.parse(JSON.toJSONString(vo)));
 	}
 
 	public void testForIssue2() {
