diff --git a/core/src/test/java/com/pholser/junit/quickcheck/ExhaustingAGivenSetTest.java b/core/src/test/java/com/pholser/junit/quickcheck/ExhaustingAGivenSetTest.java
--- a/core/src/test/java/com/pholser/junit/quickcheck/ExhaustingAGivenSetTest.java
+++ b/core/src/test/java/com/pholser/junit/quickcheck/ExhaustingAGivenSetTest.java
@@ -91,6 +91,8 @@
     }
 
     @Test public void booleansUnmarked() throws Exception {
+        UnmarkedBooleans.iterations = 0;
+        UnmarkedBooleans.testCases = null;
         assertThat(testResult(UnmarkedBooleans.class), isSuccessful());
         assertEquals(2, UnmarkedBooleans.iterations);
         assertEquals(new HashSet<>(asList(true, false)), UnmarkedBooleans.testCases);
