diff --git a/src/main/java/com/cedarsoftware/util/reflect/ClassDescriptors.java b/src/main/java/com/cedarsoftware/util/reflect/ClassDescriptors.java
--- a/src/main/java/com/cedarsoftware/util/reflect/ClassDescriptors.java
+++ b/src/main/java/com/cedarsoftware/util/reflect/ClassDescriptors.java
@@ -106,7 +106,7 @@
         final Map<String, Method> possibleInjectors = ReflectionUtils.buildInjectorMap(c);
         final ClassDescriptorImpl descriptor = new ClassDescriptorImpl();
         final Field[] declaredFields = c.getDeclaredFields();
-
+        java.util.Arrays.sort(declaredFields, java.util.Comparator.comparing(java.lang.reflect.Field::getName));
         for (Field field : declaredFields) {
 
             boolean isKnownFilter = KnownFilteredFields.instance().isFieldFiltered(field);
