diff --git a/core/src/test/java/com/alibaba/fastjson2/issues/Issue273.java b/core/src/test/java/com/alibaba/fastjson2/issues/Issue273.java
--- a/core/src/test/java/com/alibaba/fastjson2/issues/Issue273.java
+++ b/core/src/test/java/com/alibaba/fastjson2/issues/Issue273.java
@@ -18,7 +18,7 @@
         map0.put(1, 111L);
         map0.put(2, 222L);
         String json = JSON.toJSONString(map0);
-        assertEquals("{1:111,2:222}", json);
+        assertEquals(com.alibaba.fastjson2.JSON.parse("{1:111,2:222}"), com.alibaba.fastjson2.JSON.parse(json));
 
         Type type1 = new TypeReference<HashMap<Integer, Long>>() {
         }.getType();
