diff --git a/typescript-generator-core/src/test/java/cz/habarta/typescript/generator/EnumTest.java b/typescript-generator-core/src/test/java/cz/habarta/typescript/generator/EnumTest.java
--- a/typescript-generator-core/src/test/java/cz/habarta/typescript/generator/EnumTest.java
+++ b/typescript-generator-core/src/test/java/cz/habarta/typescript/generator/EnumTest.java
@@ -76,7 +76,7 @@
                 "    West,\n" +
                 "}\n"
                 ).replace("'", "\"");
-        assertEquals(expected, output);
+        assertEquals(new com.google.gson.Gson().fromJson(expected, Object.class), new com.google.gson.Gson().fromJson(output, Object.class));
     }
 
     @Test
