diff --git a/core/src/test/java/com/alibaba/fastjson2/issues_2600/Issue2672.java b/core/src/test/java/com/alibaba/fastjson2/issues_2600/Issue2672.java
--- a/core/src/test/java/com/alibaba/fastjson2/issues_2600/Issue2672.java
+++ b/core/src/test/java/com/alibaba/fastjson2/issues_2600/Issue2672.java
@@ -16,11 +16,10 @@
         Map<String, Object> map = new HashMap<>();
         map.put("k", "v");
         map.put("selfRef", map);
-        assertEquals("{\"selfRef\":{\"$ref\":\"..\"},\"k\":\"v\"}",
-                JSON.toJSONString(map, JSONWriter.Feature.ReferenceDetection));
+        assertEquals(com.alibaba.fastjson2.JSON.parse("{\"selfRef\":{\"$ref\":\"..\"},\"k\":\"v\"}"), com.alibaba.fastjson2.JSON.parse(JSON.toJSONString(map, JSONWriter.Feature.ReferenceDetection));
 
         PropertyPreFilter propertyJsonFilter = (serializer, source, property) -> true;
         assertEquals("{\"selfRef\":{\"$ref\":\"$\"},\"k\":\"v\"}",
-                JSON.toJSONString(map, propertyJsonFilter, JSONWriter.Feature.ReferenceDetection));
+                JSON.toJSONString(map, propertyJsonFilter, JSONWriter.Feature.ReferenceDetection)));
     }
 }
