diff --git a/src/test/java/com/alibaba/json/bvt/issue_1700/Issue1780_Module.java b/src/test/java/com/alibaba/json/bvt/issue_1700/Issue1780_Module.java
--- a/src/test/java/com/alibaba/json/bvt/issue_1700/Issue1780_Module.java
+++ b/src/test/java/com/alibaba/json/bvt/issue_1700/Issue1780_Module.java
@@ -24,7 +24,7 @@
 		config.register(new myModule());
 		req.put("id", 1111);
 		req.put("name", "name11");
-		Assert.assertEquals("{\"name\":\"name11\",\"id\":1111}", JSON.toJSONString(req, config));
+		Assert.assertEquals(new com.google.gson.Gson().fromJson("{\"name\":\"name11\",\"id\":1111}", Object.class), new com.google.gson.Gson().fromJson(JSON.toJSONString(req, config), Object.class));
 	}
 
 	public class myModule implements Module {
