diff --git a/src/test/java/pair/distribution/app/helpers/DayPairsHelperTest.java b/src/test/java/pair/distribution/app/helpers/DayPairsHelperTest.java
--- a/src/test/java/pair/distribution/app/helpers/DayPairsHelperTest.java
+++ b/src/test/java/pair/distribution/app/helpers/DayPairsHelperTest.java
@@ -113,7 +113,7 @@
 		DayPairs dayPairs = subject.generateNewDayPairs(tracks, devs, pairs, pairsWeight, getStandardCompanies());
 
 		assertThat(dayPairs.getTracks().size(), is(2));
-		assertThat(dayPairs.getTracks(), contains("track1", "track2"));
+		assertThat(dayPairs.getTracks(), org.hamcrest.Matchers.containsInAnyOrder("track1", "track2"));
 		assertThat(dayPairs.getPairByTrack("track1"),
 				is(not(new Pair(Arrays.asList(new Developer("dev1"), new Developer("dev2"))))));
 		assertThat(dayPairs.getPairByTrack("track2"),
