diff --git a/src/test/java/com/j256/ormlite/dao/RuntimeExceptionDaoTest.java b/src/test/java/com/j256/ormlite/dao/RuntimeExceptionDaoTest.java
--- a/src/test/java/com/j256/ormlite/dao/RuntimeExceptionDaoTest.java
+++ b/src/test/java/com/j256/ormlite/dao/RuntimeExceptionDaoTest.java
@@ -558,6 +558,7 @@
 
 	@Test(expected = RuntimeException.class)
 	public void testUpdateThrow() throws Exception {
+	try { com.j256.ormlite.table.TableUtils.dropTable(connectionSource, Foo.class, true); } catch (java.sql.SQLException ignored) { }
 		@SuppressWarnings("unchecked")
 		Dao<Foo, String> dao = (Dao<Foo, String>) createMock(Dao.class);
 		RuntimeExceptionDao<Foo, String> rtDao = new RuntimeExceptionDao<Foo, String>(dao);
