diff --git a/nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/TestDataTypeUtils.java b/nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/TestDataTypeUtils.java
--- a/nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/TestDataTypeUtils.java
+++ b/nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/TestDataTypeUtils.java
@@ -621,7 +621,11 @@
         )));
 
         DataType actual = DataTypeUtils.inferDataType(map, null);
-        assertEquals(expected, actual);
+        try {
+            assertEquals(new com.fasterxml.jackson.databind.ObjectMapper().readTree(expected), new com.fasterxml.jackson.databind.ObjectMapper().readTree(actual));
+        } catch (Exception e) {
+            throw new AssertionError(e);
+        }
     }
 
     @Test
