diff --git a/core/src/test/java/com/pholser/junit/quickcheck/SamplingOnlyFromAGivenSetTest.java b/core/src/test/java/com/pholser/junit/quickcheck/SamplingOnlyFromAGivenSetTest.java
--- a/core/src/test/java/com/pholser/junit/quickcheck/SamplingOnlyFromAGivenSetTest.java
+++ b/core/src/test/java/com/pholser/junit/quickcheck/SamplingOnlyFromAGivenSetTest.java
@@ -401,6 +401,7 @@
     }
 
     @Test public void favorValueOf() throws Exception {
+        FavorValueOf.iterations = 0;
         assertThat(testResult(FavorValueOf.class), isSuccessful());
         assertEquals(defaultPropertyTrialCount(), FavorValueOf.iterations);
     }
