diff --git a/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/AbstractTest.java b/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/AbstractTest.java
--- a/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/AbstractTest.java
+++ b/innodb-java-reader/src/test/java/com/alibaba/innodb/java/reader/AbstractTest.java
@@ -502,6 +502,7 @@
   protected static int getFieldOrdinal(Class<?> clazz, String fieldName) {
     int ordinal = 0;
     Field[] fields = clazz.getDeclaredFields();
+    java.util.Arrays.sort(fields, java.util.Comparator.comparing(java.lang.reflect.Field::getName));
     for (Field field : fields) {
       if (field.getName().equals(fieldName)) {
         return ordinal;
