diff --git a/src/test/java/com/j256/ormlite/table/SchemaUtilsTest.java b/src/test/java/com/j256/ormlite/table/SchemaUtilsTest.java
--- a/src/test/java/com/j256/ormlite/table/SchemaUtilsTest.java
+++ b/src/test/java/com/j256/ormlite/table/SchemaUtilsTest.java
@@ -32,6 +32,7 @@
 	public void testCreateSchemaName() throws SQLException {
 		SchemaUtils.dropSchema(connectionSource, SCHEMA_NAME, true);
 		assertTrue(SchemaUtils.createSchema(connectionSource, SCHEMA_NAME) > 0);
+		SchemaUtils.dropSchema(connectionSource, Schema.class, true);
 	}
 
 	@Test
