diff --git a/src/test/java/com/j256/ormlite/table/TableUtilsTest.java b/src/test/java/com/j256/ormlite/table/TableUtilsTest.java
--- a/src/test/java/com/j256/ormlite/table/TableUtilsTest.java
+++ b/src/test/java/com/j256/ormlite/table/TableUtilsTest.java
@@ -41,6 +41,7 @@
 	public void testConstructor() throws Exception {
 		@SuppressWarnings("rawtypes")
 		Constructor[] constructors = TableUtils.class.getDeclaredConstructors();
+		java.util.Arrays.sort(constructors, java.util.Comparator.comparing(java.lang.reflect.Constructor::getName));
 		assertEquals(1, constructors.length);
 		constructors[0].setAccessible(true);
 		constructors[0].newInstance();
